test/dev/ui/spinner_test.rb

```diff
@@ -60,19 +89,11 @@ def match_lines(out, *patterns)
         # newline, or cursor-down
         lines = out.split(/\n|\x1b\[\d*B/)
 
-        # Assert all lines are matched
-        lines.each do |l|
```

Comment:
This asserts that all lines of output are matched, e.g. we dont output extra stuff.